src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -243,12 +246,12 @@ class FacetSidebar extends React.Component {
                     >
                       <input
                         id={`${field}-${valueLabel}`}
-                        aria-labelledby={`${field} ${valueLabel}`}
+                        aria-labelledby={`${field}-${valueLabel}`}
```

Comment:
Oh, I thought you were seeking to have them be the same. Also I wasn't even looking at this issue -- this issue was about whether or not the facets themselves open up, not the values.